runelite-client/src/main/java/net/runelite/client/plugins/poh/BurnerOverlay.java

```diff
@@ -0,0 +1,113 @@
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.poh;
+
+import com.google.common.collect.Sets;
+import lombok.AccessLevel;
+import lombok.Getter;
+import net.runelite.api.Client;
+import net.runelite.api.GameObject;
+import net.runelite.api.Perspective;
+import net.runelite.api.Point;
+import net.runelite.client.ui.overlay.Overlay;
+import net.runelite.client.ui.overlay.OverlayPosition;
+import net.runelite.client.ui.overlay.OverlayUtil;
+import net.runelite.client.ui.overlay.components.TextComponent;
+
+import javax.annotation.Nullable;
+import javax.inject.Inject;
+import java.awt.Color;
+import java.awt.Dimension;
+import java.awt.Graphics2D;
+import java.util.Set;
+
+import static net.runelite.api.ObjectID.INCENSE_BURNER;
+import static net.runelite.api.ObjectID.INCENSE_BURNER_13209;
+import static net.runelite.api.ObjectID.INCENSE_BURNER_13210;
+import static net.runelite.api.ObjectID.INCENSE_BURNER_13211;
+import static net.runelite.api.ObjectID.INCENSE_BURNER_13212;
+import static net.runelite.api.ObjectID.INCENSE_BURNER_13213;
+
+public class BurnerOverlay extends Overlay
+{
+	@Getter(AccessLevel.PACKAGE)
+	private static final Set<Integer> BURNER_UNLIT = Sets.newHashSet(INCENSE_BURNER, INCENSE_BURNER_13210, INCENSE_BURNER_13212);
+	@Getter(AccessLevel.PACKAGE)
+	private static final Set<Integer> BURNER_LIT = Sets.newHashSet(INCENSE_BURNER_13209, INCENSE_BURNER_13211, INCENSE_BURNER_13213);
+
+	private final Client client;
+	private final PohConfig config;
+	private final PohPlugin plugin;
+	private final TextComponent textComponent = new TextComponent();
+
+	@Inject
+	public BurnerOverlay(@Nullable Client client, PohConfig config, PohPlugin plugin)
+	{
+		setPosition(OverlayPosition.DYNAMIC);
```

Comment:
After the overlay default layer changes, layer of this overlay should be set to ABOVE_SCENE